linera-core/src/unit_tests/client_tests.rs

```diff
@@ -92,8 +92,8 @@ async fn test_initiating_valid_transfer_with_notifications<B>(
 where
     B: StorageBuilder,
 {
-    let mut signer = InMemorySigner::new(None);
-    let mut builder = TestBuilder::new(storage_builder, 4, 1, &mut signer)
+    let signer = InMemorySigner::new(None);
```

Comment:
I wonder if it's something that `clippy` could help with - i.e. recognize that `InMemorySigner` is a thin wrapper for `Arc<RwLock` and `mut` is not necessary b/c it can still do internal mutation.